angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -241,6 +241,20 @@ export class IonRouterOutlet implements OnDestroy, OnInit {
     return active ? active.url : undefined;
   }
 
+  /**
+   * Returns the RouteView of the active page of each stack.
+   */
+  getLastRouteView(stackId?: string): RouteView | undefined {
+    return this.stackCtrl.getLastUrl(stackId);
+  }
+
+  /**
+   * Returns the root view in the tab stack.
+   */
+  getRootView(stackId?: string): RouteView | undefined {
```

Comment:
Mark as private

Reply:
Similarly, i have added `@internal` as it used by [ion-tabs.ts](https://github.com/ionic-team/ionic/pull/18493/files#diff-891ce68da9d3204cb1a1870a225e5350R94)